src/main/java/ua/ukr/net/Dao/AbstractJdbcDao.java

```diff
@@ -0,0 +1,5 @@
+package ua.ukr.net.Dao;
```

Comment:
Rename please package Dao to dao. Package name should start with lower case letter